pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1991,6 +2038,35 @@
       ^-  ?
       &(=(item-binding binding) =(item-duct duct))
     ==
+  ::  +add-cache: remember a cache mapping
+  ::
+  ++  add-cache
+    |=  [=path =cache-entry]
```

Comment:
Note that this trie is meant to be referentially transparent, so we desire to keep the history of all previous cache-entries in this trie slot, but only allow scrying out the most recently added one for now.

Reply:
ah. Would a `jar` be more appropriate than `axal`, then?

Reply:
Nope, We desire an `+axal` here for fast traversal, likely with a `(list)` inside where each element contains its version number and the appropriate data.